docs/manual/howtos/waitset.rst

```diff
@@ -12,36 +12,38 @@
 Waitsets
 ========
 
-Waitsets are a tool in CycloneDDS to hold execution of the program until a specific condition is reached, or an amount of time has expired.
-Waitsets and listeners cover two different requirement sets.
-Whereas Listeners can only react to changes in state, a Waitset also allows the code to react when nothing changes.
-On the other hand, Listeners allow instant callbacks to specific changes.
+The Waitsets tool in CycloneDDS holds execution of the program until a specific condition is reached, or an amount of time has expired.
+Waitsets and listeners have two different requirement sets:
+
+- A Waitset  allows the code to react when nothing changes.
```

Comment:
Better:
"A Waitset _also_ allows the code to react when nothing changes."

Reply:
Agreed. "also" added.